helm-charts/FATE-Exchange/templates/traffic-server-module.yaml

```diff
@@ -101,9 +101,15 @@ spec:
               subPath: proxy.key.pem
       {{- with .Values.modules.trafficServer.nodeSelector }}
       nodeSelector:
-      {{- range $k,$v := . }}
-        {{ $k }}: {{ $v }}
+{{ toYaml . | indent 8 }}
```

Comment:
can `nodeSelector` still work?

Reply:
Yes.

You can find similar code in other yaml. Use `toYaml` function to generate yaml content.